packages/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,285 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import Tab, { TabProps } from './Tab';
+import TabTitle from './TabTitle';
+
+const listStyle = css`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  list-style: none;
+  padding: 0px;
+  margin: 20px 0px 5px 0;
```

Comment:
Prefer not setting a margin at all for values that would be `0px`. Use `margin-top` and `margin-bottom` instead.

Reply:
Unless those margins are set by default, of course.

Reply:
👍